[BUG]Crash on apply

**Describe the bug**
I'm just testing out the provider and it crashed during my first experiment.


**To Reproduce**
Create main.tf with this content:
```
data "slack_user" "robert" {
  email = "robert@empi.re"
}

resource "slack_channel" "dev1-alerts" {
  name       = "dev1-alerts"
  is_private = false
  topic      = "Alerts for the dev1 environment"
  purpose    = "Surface interesting events for the dev1 environment."
  members = [
    data.slack_user.robert.id
  ]
}
```
Initialize:
```
Initializing the backend...
Initializing modules...

Initializing provider plugins...
- Reusing previous version of hashicorp/random from the dependency lock file
- Reusing previous version of hashicorp/aws from the dependency lock file
- Reusing previous version of gfnogueira/slack from the dependency lock file
- Using previously-installed hashicorp/random v3.7.2
- Using previously-installed hashicorp/aws v5.100.0
- Using previously-installed gfnogueira/slack v0.1.7
```
Plan:
```
random_pet.name[0]: Refreshing state... [id=willing-redbird]
module.this.data.slack_user.robert: Reading...
module.this.data.slack_user.robert: Read complete after 0s [id=U04M1BELMJS]

Terraform used the selected providers to generate the following execution plan. Resource actions are indicated with the following symbols:
  + create

Terraform will perform the following actions:

  # module.this.slack_channel.dev1-alerts will be created
  + resource "slack_channel" "dev1-alerts" {
      + id         = (known after apply)
      + is_private = false
      + members    = [
          + "U04M1BELMJS",
        ]
      + name       = "dev1-alerts"
      + purpose    = "Surface interesting events for the dev1 environment."
      + topic      = "Alerts for the dev1 environment"
    }

Plan: 1 to add, 0 to change, 0 to destroy.

Changes to Outputs:
  + result = {
      + channel = {
          + id         = (known after apply)
          + is_private = false
          + members    = [
              + "U04M1BELMJS",
            ]
          + name       = "dev1-alerts"
          + purpose    = "Surface interesting events for the dev1 environment."
          + topic      = "Alerts for the dev1 environment"
        }
    }
```
Apply:
```
module.this.slack_channel.dev1-alerts: Creating...
╷
│ Error: Request cancelled
│ 
│   with module.this.slack_channel.dev1-alerts,
│   on ../../main.tf line 11, in resource "slack_channel" "dev1-alerts":
│   11: resource "slack_channel" "dev1-alerts" {
│ 
│ The plugin.(*GRPCProvider).ApplyResourceChange request was cancelled.
╵

Stack trace from the terraform-provider-slack plugin:

panic: interface conversion: interface {} is *schema.Set, not []interface {}

goroutine 12 [running]:
github.com/gfnogueira/terraform-provider-slack/slack.resourceSlackChannelCreate({0xac09c0, 0x4000226930}, 0x4000338bd0, {0x95ffe0?, 0x40001b1280})
        github.com/gfnogueira/terraform-provider-slack/slack/resource_channel_create.go:21 +0x9c8
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).create(0x40003ac100, {0xac0918, 0x4000688d50}, 0x4000338bd0, {0x95ffe0, 0x40001b1280})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.37.0/helper/schema/resource.go:849 +0xe4
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*Resource).Apply(0x40003ac100, {0xac0918, 0x4000688d50}, 0x40001e15f0, 0x4000338ab0, {0x95ffe0, 0x40001b1280})
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.37.0/helper/schema/resource.go:980 +0x8e0
github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema.(*GRPCProviderServer).ApplyResourceChange(0x400044d290, {0xac0918?, 0x4000688c90?}, 0x40000a69b0)
        github.com/hashicorp/terraform-plugin-sdk/v2@v2.37.0/helper/schema/grpc_provider.go:1484 +0xd88
github.com/hashicorp/terraform-plugin-go/tfprotov5/tf5server.(*server).ApplyResourceChange(0x40000b0500, {0xac0918?, 0x40006881e0?}, 0x4000692080)
        github.com/hashicorp/terraform-plugin-go@v0.28.0/tfprotov5/tf5server/server.go:941 +0x2a8
github.com/hashicorp/terraform-plugin-go/tfprotov5/internal/tfplugin5._Provider_ApplyResourceChange_Handler({0x943440, 0x40000b0500}, {0xac0918, 0x40006881e0}, 0x4000692000, 0x0)
        github.com/hashicorp/terraform-plugin-go@v0.28.0/tfprotov5/internal/tfplugin5/tfplugin5_grpc.pb.go:687 +0x1c0
google.golang.org/grpc.(*Server).processUnaryRPC(0x40000d0200, {0xac0918, 0x4000688150}, 0x4000300540, 0x4000451ad0, 0x1081a08, 0x0)
        google.golang.org/grpc@v1.72.1/server.go:1405 +0xca8
google.golang.org/grpc.(*Server).handleStream(0x40000d0200, {0xac10d0, 0x40004fe000}, 0x4000300540)
        google.golang.org/grpc@v1.72.1/server.go:1815 +0x910
google.golang.org/grpc.(*Server).serveStreams.func2.1()
        google.golang.org/grpc@v1.72.1/server.go:1035 +0x84
created by google.golang.org/grpc.(*Server).serveStreams.func2 in goroutine 35
        google.golang.org/grpc@v1.72.1/server.go:1046 +0x13c

Error: The terraform-provider-slack plugin crashed!

This is always indicative of a bug within the plugin. It would be immensely
helpful if you could report the crash with the plugin's maintainers so that it
can be fixed. The output above should help diagnose the issue.
```

**Expected behavior**
Slack channel created

**Environment (please complete the following information):**
 - OS: Linux 89ff3d49ed4a 6.12.5-linuxkit #1 SMP Tue Jan 21 10:23:32 UTC 2025 aarch64 aarch64 aarch64 GNU/Linux 
 - Terraform Version: Terraform v1.8.5
 - Provider Version: provider[registry.terraform.io/gfnogueira/slack] 0.1.7

**Additional context**
It's documentation has a list of required scopes, however channels:write doesn't appear in the list of scopes in the Slack admin UI, so it may be an issue with missing scope.

--- a/slack/resource_channel_create.go
+++ b/slack/resource_channel_create.go
@@ -18,7 +18,6 @@ func resourceSlackChannelCreate(ctx context.Context, d *schema.ResourceData, met
 	isPrivate := d.Get("is_private").(bool)
 
 	// Parse members
-	//membersRaw := d.Get("members").([]interface{})
 	membersRaw := d.Get("members").(*schema.Set).List()
 	var members []string
 	for _, m := range membersRaw {

--- a/slack/resource_channel_create.go
+++ b/slack/resource_channel_create.go
@@ -18,7 +18,8 @@ func resourceSlackChannelCreate(ctx context.Context, d *schema.ResourceData, met
 	isPrivate := d.Get("is_private").(bool)
 
 	// Parse members
-	membersRaw := d.Get("members").([]interface{})
+	//membersRaw := d.Get("members").([]interface{})
+	membersRaw := d.Get("members").(*schema.Set).List()
 	var members []string
 	for _, m := range membersRaw {
 		members = append(members, m.(string))

